source/level.js

```diff
@@ -10,22 +10,27 @@ export class Level extends Phaser.Scene{
     background;
     minimap;
     timeDelta = 0;
-    // Money
-	gold = 0;
-	goldText = '';
     // Character
-	characterClass = '';
+    characterData = {
+        gold: 0,
+        characterClass: '',
+        enemiesKilled: 0,
+        timesClicked: 0,
+        damageByClicking: 0,
+        damageByAutoClick: 0,
+        numberOfAutoClickers: 0,
+        levelCompletions: {
```

Comment:
K, with this change when you complete the level quest it unlocks the next level rather than your own. For example tutorial island unlocks lumbridge, lumbridge unlocks varrock etc.
For now I'm going to set tutorial island and lumbridge to unlocked until tutorial island is created